shell/edit/provisioning.cattle.io.cluster/index.vue

```diff
@@ -355,6 +351,20 @@ export default {
   },
 
   methods: {
+    showRkeToggle(groupedSubTypes, i) {
+      if (this.isImport || !this.rke2Enabled) {
+        return false;
+      }
+      const nodeDriverItem = groupedSubTypes.findIndex(obj => [_RKE1, _RKE2].includes(obj.name));
+      const customClusterItem = groupedSubTypes.findIndex(obj => ['custom', 'custom1', 'custom2'].includes(obj.name));
```

Comment:
This is more of a note for something to consider; we can make `nodeDriverItem` and `CustomClusterItem` computed props instead of finding the index each time `showRkeToggle` is invoked. I don't foresee the number of grouped subtypes growing to become exceedingly large, so the impact of making such a change is low.